index.html

```diff
@@ -1925,6 +1925,18 @@ <h4>Error Responses</h4>
           </span>
         </p>
       </section>
+
+      <section id="security">
+        <h4 id="security">Transport security</h4>
+      </p>
+      <p class="ednote">
+        Consider mandating HTTPs, at least for things that are used outside of a closed network.
+        Some other standards, such as Echonet use HTTPS only.
```

Comment:
nit:
* once "HTTPs" while the second time "HTTPS" with capital S letter
* ECHONET seems to be always capitalized

Reply:
👍